deploy-core.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -e
+
+[[ -n "$FOUNDRY_ROOT_CHAINID" ]] || {
+    [[ -n $ETH_RPC_URL ]] || {
+        echo "Please set FOUNDRY_ROOT_CHAINID (1 or 5) or ETH_RPC_URL";
+        exit 1;
+    }
+    export FOUNDRY_ROOT_CHAINID="$(cast chain-id)"
+}
+[[ "$FOUNDRY_ROOT_CHAINID" == "1" ]] || [[ "$FOUNDRY_ROOT_CHAINID" == "5" ]] || {
+    echo "Invalid chainid of $FOUNDRY_ROOT_CHAINID. Please set your forking environment via ETH_RPC_URL or manually by defining FOUNDRY_ROOT_CHAINID (1 or 5)."
+    exit 1;
+}
+
+[[ "$FOUNDRY_ROOT_CHAINID" == "1" ]] && echo "Deploying D3M Core on Mainnet"
+[[ "$FOUNDRY_ROOT_CHAINID" == "5" ]] && echo "Deploying D3M Core on Goerli"
+
+rm -f out/contract-exports.env
+export FOUNDRY_ROOT_CHAINID
+forge script script/D3MCoreDeploy.s.sol:D3MCoreDeployScript --use solc:0.8.14 --rpc-url $ETH_RPC_URL --sender $ETH_FROM --broadcast --verify
```

Comment:
```suggestion
    FOUNDRY_ROOT_CHAINID="$(cast chain-id)"
}
[[ "$FOUNDRY_ROOT_CHAINID" == "1" ]] || [[ "$FOUNDRY_ROOT_CHAINID" == "5" ]] || {
    echo "Invalid chainid of $FOUNDRY_ROOT_CHAINID. Please set your forking environment via ETH_RPC_URL or manually by defining FOUNDRY_ROOT_CHAINID (1 or 5)."
    exit 1;
}

[[ "$FOUNDRY_ROOT_CHAINID" == "1" ]] && echo "Deploying D3M Core on Mainnet"
[[ "$FOUNDRY_ROOT_CHAINID" == "5" ]] && echo "Deploying D3M Core on Goerli"

export FOUNDRY_ROOT_CHAINID
rm -f out/contract-exports.env
export FOUNDRY_ROOT_CHAINID
forge script script/D3MCoreDeploy.s.sol:D3MCoreDeployScript --use solc:0.8.14 --rpc-url "$ETH_RPC_URL" --sender "$ETH_FROM" --broadcast --verify
```

Fixes shellcheck issues and leaves the same format than the test file of dss-test